NRediSearch/Query.cs

```diff
@@ -162,6 +162,16 @@ public Paging(int offset, int count)
         public byte[] Payload { get; set; }
 
         /// <summary>
+        /// Set the query parameter to sort by
+        /// </summary>
+        public string _sortBy { get; set; }
```

Comment:
naming: this is not idiomatic .NET naming conventions for a property - should be `SortBy`